Sources/Plasma/FeatureLib/pfPython/cyMisc.cpp

```diff
@@ -1090,34 +1067,27 @@ std::vector<PyObject*> cyMisc::GetPlayerListDistanceSorted()
     std::vector<PyObject*> pyPL;
 
     // get the sorted member list from the Net transport manager
-    plNetTransportMember **members = nullptr;
-    plNetClientMgr::GetInstance()->TransportMgr().GetMemberListDistSorted( members );
-    if (members != nullptr)
+    std::vector<plNetTransportMember*> members = plNetClientMgr::GetInstance()->TransportMgr().GetMemberListDistSorted();
+    for (plNetTransportMember* mbr : members)
     {
-        int i;
-        for( i = 0; i < plNetClientMgr::GetInstance()->TransportMgr().GetNumMembers(); i++ )
+        plKey avkey = mbr->GetAvatarKey();
+        if (avkey)
         {
-            plNetTransportMember *mbr = members[ i ];
-            plKey avkey = mbr->GetAvatarKey();
-            if (avkey)
+            // only non-ignored people in list and not in ignore list
+            if (!VaultAmIgnoringPlayer(mbr->GetPlayerID()))
             {
-                // only non-ignored people in list and not in ignore list
-                if ( !VaultAmIgnoringPlayer ( mbr->GetPlayerID()) )
-                {
-                    PyObject* playerObj = pyPlayer::New(avkey, mbr->GetPlayerName().c_str(), mbr->GetPlayerID(), mbr->GetDistSq());
-                    pyPlayer* player = pyPlayer::ConvertFrom(playerObj); // accesses internal pyPlayer object
+                PyObject* playerObj = pyPlayer::New(avkey, mbr->GetPlayerName().c_str(), mbr->GetPlayerID(), mbr->GetDistSq());
```

Comment:
```suggestion
                PyObject* playerObj = pyPlayer::New(avkey, mbr->GetPlayerName(), mbr->GetPlayerID(), mbr->GetDistSq());
```

Spin washing an `ST::string`... Is there a way we could write some kind of automated check for this?